scripts/runlogwatch.sh

```diff
@@ -5,9 +5,12 @@ LOG_DIR="/shared/log/ironic/deploy"
 
 mkdir -p "${LOG_DIR}"
 
+# shellcheck disable=SC2034
 python3 -m pyinotify -e IN_CLOSE_WRITE -v "${LOG_DIR}" |
-    while read -r path _action file; do
-        echo "************ Contents of ${path}/${file} ramdisk log file bundle **************"
-        tar -xOzvvf "${path}/${file}" | sed -e "s/^/${file}: /"
-        rm -f "${path}/${file}"
+    while read -r event dir mask maskname filename filepath pathname wd; do
+        FILENAME=$(echo "$filename" | cut -d'=' -f2- | sed -e 's/\x1b\[[0-9;]*m//g')
```

Comment:
```suggestion
        FILENAME=$(echo "${filename}" | cut -d'=' -f2- | sed -e 's/\x1b\[[0-9;]*m//g')
```
The above comment would also help understand what are we sed'ing here.